server/app/api/user/user_controller.py

```diff
@@ -39,4 +39,10 @@ def test(jwt: JWTDep):
         { "id": 1, "name": "Bub" },
         { "id": 2, "name": "Coc" },
         { "id": 3, "name": "Did" },
-    ]
\ No newline at end of file
+    ]
+
+@router.get("/search/{query}", status_code=200, response_model=List[UserDTO], summary="Search users by username prefix")
+@pre_authorize([UserRole.user, UserRole.admin])
+def search_by_username_prefix(jwt: JWTDep, query: str, organization_id: int = 0, user_service: UserService = Depends(get_user_service)):
+    users = user_service.search_by_username_prefix(query, organization_id)
+    return users
```

Comment:
`organization_id` should be renamed to better explain what the field does, like `org_id_whose_members_to_exclude` (just like in `user.api.ts`).